src/mixins/filing-template-mixin.ts

```diff
@@ -209,7 +209,7 @@ export default class FilingTemplateMixin extends DateMixin {
       if (this.getHasPlanOfArrangement || this.getFileNumber) {
         filing.correction.courtOrder = {
           fileNumber: this.getFileNumber,
-          effectOfOrder: this.getHasPlanOfArrangement ? EffectOfOrders.PLAN_OF_ARRANGEMENT : null,
+          effectOfOrder: this.getHasPlanOfArrangement ? EffectOfOrders.PLAN_OF_ARRANGEMENT : undefined,
```

Comment:
Please verify by filing a correction with a court order number and with / without a POA.

Reply:
Tested ok with CP1002614